backend/CircleForms/Models/Result.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Net;
+
+namespace CircleForms.Models;
+
+public class Result<T>
+{
+    public Result(T value)
+    {
+        Value = value;
+        IsError = false;
+    }
+
+    public Result(HttpStatusCode code, string message)
+    {
+        StatusCode = code;
+        Message = message;
+        IsError = true;
+    }
+
+    public Result(string message)
+    {
+        StatusCode = HttpStatusCode.BadRequest;
```

Comment:
`HttpStatusCode.InternalServerError`? I think if we know it's something related to requests being invalid, we'd know status code as well.

Reply:
This overload is specific to BadRequest messages, if you want to InternalServerError then throw an exception

Reply:
Then do it similarly to `public static Result<T> NotFound(string id)`? Not sure why we should have three different ways of creating a result